### ISSUE TITLE
Can I use the Email app as the frontend for my private VPS email server?

### ISSUE THREAD

**Issue by SuperPauly** [ProtonMail/WebClients#390]
> We are happy to answer your questions about the code or discuss technical ideas.

Please complete the following checklist (by adding [x]):

-   [Y] I have searched open and closed issues for duplicates
-   [Y] This isn't a feature request
-   [Y] This is not a report about my app not working as expected

---

I can't find a direct answer to this question. It's license is GPL but doesn't mean it's going to work with Emails servers that are not official Proton Mail servers. 

So I just need a short and simple yes or no answer.

Which apps can I use that will not be restricted if I use open source tools to connect them to? For example.

Can I use Mail app with Postfix/Sendmail/Exa?
Can I use the Drive app and connect it to a Samba/NFS/Other network file system mount?
Can I use the Calendar app and sync it with my own calendar I've made which the data is stored on a PostgreSQL instance?

I'm just a bit confused by the usage & setup not being defined to clearly in the readme, and I've lost 1 week of work because of a situation similar with Tutanota, They like to claim there open source but there open source web client only works with there proprietary servers and it's pain to back engineer. 

I hope this isn't a similar repo.

Thankyou.

**Comment by thgoebel** [comment]
No. All of their code is tightly coupled to the Proton ecosystem, you would need to make some engineering effort to make it work. E.g. their email clients retrieve email via Proton's API and not via IMAP (afaik).

This is why F-Droid has the ["Non-Free Network" anti-feature](https://f-droid.org/en/docs/Anti-Features/#NonFreeNet): it refers to network services that are either not open source, or cannot be changed. (Though there are plans to split this up into a dedicated ["Tethered Net"](https://github.com/f-droid/fdroiddata/blob/de6b369206961272244e73c7a966cde8c59d3e3f/config/antiFeatures.yml#L40-L42) anti-feature to make the distinction clearer.)

> They like to claim there open source

Yeah, unfortunately, "open source" means just that: the source code is publicly visible. It does not mean "freely licensed" or "easily usable".

Proton is not alone here: Signal or Threema are also open source, but tied to their backends.

And at Proton, "open source" also means:
- "with a good amount of delay". E.g. the last commit to main in this repo is 3 weeks ago, and the last tagged proton-mail version is 5.0.38.4 but the deployed version is 5.0.40.2.
- "or not open source at all". E.g. the Proton Calendar mobile apps are still not Open Source, despite e.g. the Android app having been in beta [since December 2020](https://proton.me/blog/calendar-android-beta) and out of beta since [April 2022](https://proton.me/blog/calendar-android-app).
